Source/WebCore/page/ScreenOrientationLockType.h

```diff
@@ -48,13 +49,14 @@ template<> struct EnumTraits<WebCore::ScreenOrientationLockType> {
     using values = EnumValues<
         WebCore::ScreenOrientationLockType,
         WebCore::ScreenOrientationLockType::Any,
-        WebCore::ScreenOrientationLockType::Natural,
         WebCore::ScreenOrientationLockType::Landscape,
+        WebCore::ScreenOrientationLockType::LandscapePrimary,
```

Comment:
Why are these enum traits still here if you are also adding the enum to the serialization.in. As I mentioned in a previous review, it is one of the other.

also the changes to this enum traits don’t seem right. As far as I know, the ordering needs to match the enum’s ordering.

Reply:
Sorry about that! Not sure what happened there. I thought I deleted these :/